### ISSUE TITLE
Suggestion to add accessibility section to the MDN article for the “datalist” element

### ISSUE THREAD

**Issue by Mottie** [mdn/content#22958]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist

### What specific section or headline is this issue about?

None

### What information was incorrect, unhelpful, or incomplete?

Our accessibility expert told me that we are not able to use a `datalist` because of these issues:
- It doesn't zoom along with the browser
- And it cannot be styled for high-contrast mode
- NVDA + Firefox doesn't announce anything when the autosuggest popup opens

I think this would be valuable information to include on the page!

### What did you expect to see?

A new "accessibility concerns" section. It'd be awesome to add this to _all_ pages eventually.

### Do you have any supporting links, references, or citations?

[Testing typeaheads presentation](https://docs.google.com/presentation/d/181nGvo3LS_eGzSpTCNciyv9YJ2g_ATYjbVIRecxh5zY/edit?usp=sharing)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/datalist`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/datalist/index.md
* Last commit: https://github.com/mdn/content/commit/0f2d4cfa06ad3c58f07a7be525188ac14d83603d
* Document last modified: 2022-12-11T07:52:13.000Z

</details>

**Comment by CBID2** [comment]
Hi @sideshowbarker and @Mottie! :) Here's my PR: #27931

**Comment by Mottie** [comment]
Thank you! cc @joshkimux
